src/boost_python_exception/python_compat.hpp

```diff
@@ -0,0 +1,16 @@
+#ifndef INCLUDE_BOOST_PYTHON_EXCEPTION_PYTHON_COMPAT_HPP
+#define INCLUDE_BOOST_PYTHON_EXCEPTION_PYTHON_COMPAT_HPP
+
+/* Python 2/3 compatibility layer. */
+
+#include <Python.h>
+#include <string>
+
+namespace boost_python_exception {
+
+std::string python_string_as_std_string(PyObject*);
```

Comment:
a little documentation would be nice, even though it's pretty self-explanatory...